packages/boxel-ui/addon/src/components/resizable-panel-group/index.gts

```diff
@@ -253,73 +298,83 @@ export default class ResizablePanelGroup extends Component<Signature> {
   onWindowResize(entry: ResizeObserverEntry, _observer: ResizeObserver) {
     let panelGroupEl = entry.target as HTMLElement;
 
-    let panelWidths = [];
+    let panelLengths = [];
     for (let index = 1; index <= this.listPanelContext.size; index++) {
       let panelEl = panelGroupEl.querySelector(
-        `[id='${index}'].boxel-panel`,
+        `[id='${index}'].boxel-panel-${this.args.orientation}`,
       ) as HTMLElement;
       if (!panelEl) {
         console.error(
           `Could not find selector: [id='${index}'].boxel-panel when handling window resize for resizeable panel group`,
         );
         continue;
       }
-      panelWidths.push(panelEl.offsetWidth);
+      panelLengths.push(panelEl[this.offsetLengthProperty]);
     }
-    let totalPanelWidth = panelWidths.reduce((partialSum, a) => partialSum + a);
+    let totalPanelLength = panelLengths.reduce(
+      (partialSum, a) => partialSum + a,
+    );
+    let resizeHandlerSelector = `#resize-handler-${this.args.orientation}-1`;
     let resizeHandlerEl = panelGroupEl.querySelector(
-      '#resize-handler-1',
+      resizeHandlerSelector,
     ) as HTMLElement;
     if (!resizeHandlerEl) {
       console.error(
-        `Could not find selector: #resize-handler when handling window resize for resizeable panel group`,
+        `Could not find selector: ${resizeHandlerSelector} when handling window resize for resizeable panel group`,
       );
       return;
     }
 
-    let resizeHandlerWidth = (resizeHandlerEl as HTMLElement).offsetWidth;
-    let totalResizeHandlerWidth =
-      resizeHandlerWidth * (this.listPanelContext.size - 1);
+    let resizeHandleContainer = (resizeHandlerEl as HTMLElement).parentElement!;
+    let resizeHandlerLength = resizeHandleContainer[this.offsetLengthProperty];
```

Comment:
This is a slight departure in implementation, the padding of the handle container element needed to be included in the length calculation to fix the problem where resizing the left column caused its internal ratios to change, here’s what happened before this:

![screencast 2023-10-18 17-24-50](https://github.com/cardstack/boxel/assets/43280/b581a32a-0dfe-4957-b82d-8ae0e097077f)